Python/instrumentation.c

```diff
@@ -891,18 +891,39 @@ static inline int most_significant_bit(uint8_t bits) {
 static uint32_t
 global_version(PyInterpreterState *interp)
 {
-    return interp->ceval.eval_breaker & ~_PY_EVAL_EVENTS_MASK;
+    return interp->ceval.interp_eval_breaker & ~_PY_EVAL_EVENTS_MASK;
 }
 
 static void
-set_global_version(PyInterpreterState *interp, uint32_t version)
+set_version_raw(uintptr_t *breaker, uint32_t version)
 {
-    assert((version & _PY_EVAL_EVENTS_MASK) == 0);
-    uintptr_t old = _Py_atomic_load_uintptr(&interp->ceval.eval_breaker);
+    uintptr_t old = _Py_atomic_load_uintptr(breaker);
     intptr_t new;
```

Comment:
Predates your PR, but this looks like it should be `uintptr_t`:

```suggestion
    uintptr_t new;
```